.github/workflows/dogfood-gitops.yml

```diff
@@ -76,3 +76,32 @@ jobs:
           DOGFOOD_COMPANY_OWNED_IPHONES_ENROLL_SECRET: ${{ secrets.DOGFOOD_COMPANY_OWNED_IPHONES_ENROLL_SECRET }}
           DOGFOOD_COMPANY_OWNED_IPADS_ENROLL_SECRET: ${{ secrets.DOGFOOD_COMPANY_OWNED_IPADS_ENROLL_SECRET }}
           MANAGED_CHROME_ENROLLMENT_TOKEN: ${{ secrets.CLOUD_MANAGEMENT_ENROLLMENT_TOKEN }}
+
+      - name: Notify on Gitops failure
+        if: failure() && github.ref_name == 'main'
+        uses: slackapi/slack-github-action@e28cf165c92ffef168d23c5c9000cffc8a25e117 # v1.24.0
```

Comment:
There's newer versions but this is what we use elsewhere, and it's nice and simple.  We can look at upgrading all the uses to v2 if it becomes a concern.